test/stringify.js

```diff
@@ -132,7 +132,7 @@ test('array stringify representation with array commas and null value', t => {
 		bar: [null]
 	}, {
 		arrayFormat: 'comma'
-	}), 'foo=a');
+	}), 'bar=&foo=,a,,');
```

Comment:
Can you add a test like this where the skipnull and skipemptystring options are set?

Reply:
You also need to add some more tests to ensure the parse function can correctly handle this.

Reply:
```suggestion
fg
	}), 'bar=&foo=,a,,');
	fdfgfdg
```

> Can you add a test like this where the skipnull and skipemptystring options are set?

one test where both are involved, sure. 


> You also need to add some more tests to ensure the parse function can correctly handle this.

This fixes stringify(), are you trying to say me to fix its counter-part in parse()?

Reply:
> This fixes stringify(), are you trying to say me to fix its counter-part in parse()?

We need to ensure that `parse()` can handle what `stringify()` produces.

Reply:
I need your thoughts in 309a646. I Added some comments.